packages/flutter_tools/lib/src/commands/build_ios.dart

```diff
@@ -129,12 +130,46 @@ class BuildIOSArchiveCommand extends _BuildIOSSubCommand {
     return super.validateCommand();
   }
 
+  Future<void> _validateXcodeBuildSettingsAfterArchive() async {
+
+    final BuildableIOSApp app = await buildableIOSApp;
+
+    final String plistPath = app.builtInfoPlistPathAfterArchive;
+
+    if (!globals.fs.file(plistPath).existsSync()) {
+      globals.printError('Invalid iOS archive. Does not contain Info.plist.');
+      return;
+    }
+
+    final Map<String, String?> xcodeProjectSettingsMap = <String, String?>{};
+    
+    xcodeProjectSettingsMap['Version'] = globals.plistParser.getStringValueFromFile(plistPath, PlistParser.kCFBundleShortVersionStringKey);
+    xcodeProjectSettingsMap['Build'] = globals.plistParser.getStringValueFromFile(plistPath, PlistParser.kCFBundleVersionKey);
+    xcodeProjectSettingsMap['Display Name'] = globals.plistParser.getStringValueFromFile(plistPath, PlistParser.kCFBundleDisplayNameKey);
+    xcodeProjectSettingsMap['Deployment Target'] = globals.plistParser.getStringValueFromFile(plistPath, PlistParser.kMinimumOSVersionKey);
+    xcodeProjectSettingsMap['Bundle Identifier'] = globals.plistParser.getStringValueFromFile(plistPath, PlistParser.kCFBundleIdentifierKey);
+
+    final List<String> parsedSettings = <String>[];
+    xcodeProjectSettingsMap.forEach((String title, String? info) {
+      parsedSettings.add('$title: ${info ?? "Missing"}');
+    });
+
+    String message = parsedSettings.join('\n');
+    if (xcodeProjectSettingsMap.values.any((String? element) => element == null)) {
+      message += '\n\nYou must set up the missing settings\n';
+      message += 'Instructions: https://docs.flutter.dev/deployment/ios#review-xcode-project-settings';
```

Comment:
The version number and build number aren't set in Xcode, but I guess it's better than the highest level https://docs.flutter.dev/deployment/ios ?

Reply:
yeah neither is ideal. i'm fine with using neither links. 

What's your thought? @leighajarett?

Reply:
Using the top level link has the benefit that it's more stable. It's very possible that we update the structure of the website and have a broken link.

Reply:
I think I will just use `https://docs.flutter.dev/deployment/ios` then.